docs/platform/howto/recovering-a-deleted-service.rst

```diff
@@ -0,0 +1,14 @@
+﻿Recovering a deleted service
+============================
+
+If a service got deleted, will we be able to recover it again?
```

Comment:
I suggest we move away from a question and try directly saying what one can/can't do